src/SourceBuild/patches/wpf/0001-Add-references-needed-by-Formatters.Nrbf.patch

```diff
@@ -0,0 +1,29 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Eric StJohn <ericstj@microsoft.com>
+Date: Wed, 14 Aug 2024 10:38:31 -0700
+Subject: [PATCH] Add references needed by Formatters.Nrbf
+
```

Comment:
Please add a `Backport: <PR/issue`>` line that tracks addressing the underlying issue/removal of this patch.  See https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md#patch-guidelines

Reply:
I put that line in the commit, I didn't realize it had to go in the patch as well.   Could we not reset this PR again on adding a comment?

Reply:
We place it in the patch for UX purposes.  Patches can be updated numerous times over their lifetime.  Having the backport tracked in the patch makes is easier to track on two fronts - 1. make sure the backport is done 2. make it easy to know when the patch can be removed on dependency flow PRs.

> Could we not reset this PR again on adding a comment?

Are you asking for an exception to avoid another commit that would reset the CI process?  e.g. you want to get this in ASAP?  Given the backport PR is already open, I am willing to be flexible.

Reply:
If more changes are needed, I would ask that the patch be updated.

Reply:
I'm hoping to get this codeflow change in before repos snap for RC1 today, we're close here and it seems the rebuilds take many hours.  If we need to kick off another build I'll make sure to add the comment.

Reply:
Turns out I needed to fix up a test, so I added the comment.